scraper.py

```diff
@@ -116,6 +128,4 @@ def main():
     with open("bchydro_outages.json", "w") as outfile:
         json.dump(output, outfile, indent=2, sort_keys=True)
 
-
-if __name__ == "__main__":
-    main()
+main()
```

Comment:
```suggestion
if __name__ == "__main__":
    main()
```

I mean it doesn't make much of a difference but I'd stick with the convention.